src/components/templates/Auditorium/components/AllSectionPreviews/AllSectionPreviews.scss

```diff
@@ -17,6 +17,14 @@ $all-section-previews-bottom-spacing: 50px;
   grid-auto-rows: $section-height;
   grid-auto-columns: minmax($section-min-width, 1fr);
 
+  &__background {
```

Comment:
this should go to ContainerWithBackground, i believe.

Reply:
fixed b396ea2